tests/test_examples.sh

```diff
@@ -1,8 +1,9 @@
 # Run from the root of the project using `bash tests/test_examples.sh`
+set -e
 echo "Running test_examples.sh"
 
 echo "Running the polarify example:"
-pixi run -v --manifest-path examples/polarify/pixi.toml test
+pixi run -v --manifest-path examples/polarify/pixi.toml tes
```

Comment:
```suggestion
pixi run -v --manifest-path examples/polarify/pixi.toml test
```

:warning: Commit this after the ci failed correctly